web/hooks/organization.go

```diff
@@ -0,0 +1,47 @@
+package hooks
+
+import (
+	"context"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/quickfeed/quickfeed/qf"
+	"github.com/quickfeed/quickfeed/scm"
+)
+
+func (wh GitHubWebHook) handleOrgInvite(e *github.OrganizationEvent) {
+	// check that invitee is enrolled in course
+	invitee := e.GetInvitation().GetLogin()
+	user, err := wh.db.GetUserByCourse(&qf.Course{ScmOrganizationID: uint64(e.GetOrganization().GetID())}, invitee)
+	if err != nil {
+		wh.logger.Error("Invite: Failed to find user:", err)
```

Comment:
Elsewhere we log it like this:
```go
		user, err := wh.db.GetUserByCourse(course, login)
		if err != nil {
			wh.logger.Errorf("Failed to find user %s in course %s: %v", login, course.GetName(), err)
			return
		}
```
Suggest to prefix it with `Invite:` and replace `course.GetName()` with `course.GetScmOrganizationName()`.